Frontend/src/app/CaptureImageScreen.js

```diff
@@ -319,6 +323,27 @@ export default function CaptureImageScreen() {
 
             <EmailInput email={email} setEmail={setEmail} />
 
+            {/* Add a section for QR Code Option */}
+            <View style={{ marginVertical: spacing.md }}>
+              <TouchableOpacity
+                style={{
+                  ...GlobalStyles.optionButton,
+                  backgroundColor: qr === 'selected' ? colors.primary : colors.gray[200],
+                  padding: spacing.md,
+                  borderRadius: borderRadius.sm,
+                  alignItems: 'center',
+                }}
+                onPress={() => setQr('selected')}
+              >
+                <Text style={{ 
+                  color: qr === 'selected' ? colors.textInverse : colors.text, 
+                  ...fonts.body 
+                }}>
+                  Send via QR Code
+                </Text>
+              </TouchableOpacity>
+            </View>
+
```

Comment:
also this is how I am currently sending to the backend that qr code is selected, which is needed.